apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -346,7 +346,7 @@ static int make_long_file(void)
  * @brief            Mount file system
  * @scenario         Mount initialized file system
  * @apicovered       mount
- * @precondition     File system should be initialized. For smartfs, smart_initialize & mksmartfs should be excuted.
+ * @precondition     File system should be initialized. For jSmartFs, smart_initialize & mksmartfs should be excuted.
```

Comment:
However, I also request you to look at the "smartfs test applications" folder and make changes there and include that in this commit itself.

After that, the path in the commit title should only be "apps/examples: ...."